lib/measures/add_ems_emissions_reporting/measure.rb

```diff
@@ -446,25 +489,35 @@ def run(model, runner, usr_args)
     out_var4.setKeyValue('EMS')
     out_var4.setReportingFrequency('hourly')
 
-    # add future hourly intensity reporting output variable
-    out_var5 = OpenStudio::Model::OutputVariable.new('Future_Hourly_Emissions_Intensity_Var', model)
+    # add natural gas reporting output variable
+    out_var5 = OpenStudio::Model::OutputVariable.new('Natural_Gas_Emissions_Var', model)
     out_var5.setKeyValue('EMS')
     out_var5.setReportingFrequency('hourly')
 
-    # add historical hourly intensity reporting output variable
-    out_var6 = OpenStudio::Model::OutputVariable.new('Historical_Hourly_Emissions_Intensity_Var', model)
+    # add future hourly intensity reporting output variable
+    out_var6 = OpenStudio::Model::OutputVariable.new('Future_Hourly_Emissions_Intensity_Var', model)
     out_var6.setKeyValue('EMS')
     out_var6.setReportingFrequency('hourly')
 
-    # add future annual intensity reporting output variable
-    out_var7 = OpenStudio::Model::OutputVariable.new('Future_Annual_Emissions_Intensity_Var', model)
+    # add historical hourly intensity reporting output variable
+    out_var7 = OpenStudio::Model::OutputVariable.new('Historical_Hourly_Emissions_Intensity_Var', model)
     out_var7.setKeyValue('EMS')
     out_var7.setReportingFrequency('hourly')
 
-    # add historical annual intensity reporting output variable
-    out_var8 = OpenStudio::Model::OutputVariable.new('Historical_Annual_Emissions_Intensity_Var', model)
+    # add future annual intensity reporting output variable
+    out_var8 = OpenStudio::Model::OutputVariable.new('Future_Annual_Emissions_Intensity_Var', model)
     out_var8.setKeyValue('EMS')
     out_var8.setReportingFrequency('hourly')
+
+    # add historical annual intensity reporting output variable
+    out_var9 = OpenStudio::Model::OutputVariable.new('Historical_Annual_Emissions_Intensity_Var', model)
+    out_var9.setKeyValue('EMS')
+    out_var9.setReportingFrequency('hourly')
+
+    # add natural gas intensity reporting output variable
+    out_var10 = OpenStudio::Model::OutputVariable.new('Natural_Gas_Emissions_Intensity_Var', model)
+    out_var10.setKeyValue('EMS')
+    out_var10.setReportingFrequency('hourly')
     
```

Comment:
any reason you added the natural gas in the middle and then incremented the existing objects? it makes the diff messier, couldn't you just add the natural gas at the end? maybe there is a reason, but I was curious.

Reply:
no reason. I will replace the natural gas, propane, fuel oil to be at the end.